### PR TITLE
PP-79: CSS Code Refactoring: Color Standardization and Improved Maintainability

### PR DESCRIPTION
**Pull request by AlonaVladymyrovaTrinity**
This pull request introduces a comprehensive refactoring of the project's CSS codebase to enhance maintainability and standardize color usage. The primary focus of this update is to replace direct color values in all CSS files with references to global variables defined in the index CSS file. By centralizing color definitions, we establish a consistent color palette throughout the application, making it easier to manage and update in the future.

During the refactoring process, each CSS file was carefully analyzed to identify instances of direct color values, which were subsequently replaced with their corresponding global variables. This approach streamlines color management by enabling developers to modify a single variable to effect changes across the entire project, rather than manually updating each individual instance.

By leveraging global variables, this refactoring effort improves code readability and promotes code reuse, reducing redundancy in the CSS codebase. The standardized color usage also enhances the maintainability of the project, as future updates or adjustments to the color scheme can be implemented efficiently and consistently.

Please note that this pull request focuses exclusively on the refactoring of CSS code and does not introduce any functional changes to the application.

### CODE REVIEW

File: src/component/Home/Home.module.css [hunk 1]
```diff
@@ -53,18 +54,14 @@ video {
...
-  } */
   .home_banner_content {
     height: 5rem;
     position: relative;
     align-items: center;
     padding-left: 0;
   }
   .home_banner_content h1 {
-    color: var(--color-txt);
+    color: var(--color-white);
```
Comment (LiubovCass) [inline src/component/Home/Home.module.css]: If video {display: none} then .home_banner_content h1 {color: var(--color-txt)}. White text will be low contrast.

**Review by LiubovCass — COMMENTED** [review]
Alona, it is really accurate job! I left comment about text color in Home.module.css

**Comment by badgerbadgers** [comment]
Y'all are stepping it up with CSS styling, that's what I'm talking about this makes styling the app so much easier! Excellent writeup for the PR as well.

**Review by badgerbadgers — APPROVED** [review]
LGTM! 
There are alot of files here, in future PRs might be easier to break this PR down in manageable chunks where:
-each component or folder with CSS module is a commit
-you create a PR with 2 or 3 components max. 

These are the next level CSS practices that take your project to the next level! Great work!

File: src/component/Home/Home.module.css (hunk 1 above)
Comment (AlonaVladymyrovaTrinity) [reply src/component/Home/Home.module.css]: Сhanges made in accordance with the review recommendations.
Please review again.

**Review by LiubovCass — APPROVED** [review]

**Comment by AlonaVladymyrovaTrinity** [comment]
Thanks for the review, we try to take into account all the comments and recommendations from mentors

**Pull request merged**